src/commands/object/delete.ts

```diff
@@ -33,26 +30,12 @@ export class DeleteCommand extends Command {
                 console.log();
             }
         } else {
-            const obj = session.getObject<graphene.Storage>(Handle.toBuffer(params.oid));
-            if (!obj) {
-                throw new Error(`Object by ID '${params.oid}' is not found`);
-            }
-
-            // Print info about object
-            print_caption(`Object info`);
-            print_object_info(obj);
-            console.log();
-
-            const answer = await readline.question("Do you really want to remove this object (Y/N)? ");
-            if (answer && (answer === "yes" || answer === "y")) {
-                session.destroy(obj!);
 
-                console.log();
-                console.log("Object was successfully removed");
-                console.log();
-            }
+            const objects = session.find({id:Buffer.from(params.oid,'hex')});
+            session.destroy(objects.items(0)!);
+            session.destroy(objects.items(1)!);
+            console.log('Object deleted.');
```

Comment:
Non- interactive shouldn't have any stdOut output unless verbose option is provided.
Also, maybe add a verbose option.